<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-30</y>
      <width>431</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
background-color:#4A4A4A;

}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="AuthorisationButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
color:#141414;
background-color:#DDDAD9;
}</string>
     </property>
     <property name="text">
      <string>Authorisation</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordLabel">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>280</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font-family: &quot;Gill Sans&quot;, sans-serif;
color:#DDDAD9;
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LoginEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>133</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
background-color:#DDDAD9;

}</string>
     </property>
    </widget>
    <widget class="QLabel" name="LoginLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font-family: &quot;Gill Sans&quot;, sans-serif;
color:#DDDAD9;
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>133</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
background-color:#DDDAD9;
}</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ShowPassBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
font-family: &quot;Gill Sans&quot;, sans-serif;
color:#DDDAD9;

}</string>
     </property>
     <property name="text">
      <string>Show password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
